src/qml/components/AttachmentPreview.qml

```diff
@@ -26,7 +31,7 @@ Item {
         }
         text: qsTr("Close")
         color: "white"
-        font.bold: true
+        font.pointSize: 14
```

Comment:
Please use UiHelper.fixFontSz for font sizes

Reply:
btw, do you think it's better to replace Close label with Close icon?

Reply:
Yes, it would be better to use close icon. We can also use it without icon or label